apps/betterangels-backend/notes/types.py

```diff
@@ -287,3 +288,38 @@ class UpdateTaskLocationInput:
 class RevertNoteInput:
     id: ID
     revert_before_timestamp: datetime
+
+
+@strawberry_django.filters.filter(User)
+class InteractionAuthorFilter:
+    first_name: auto
+    last_name: auto
+    middle_name: auto
```

Comment:
you can remove these. these provide a whole word, case sensitive search. i.e., a search for "jack" or "Jac" wouldn't return a result with the name "Jack". The custom search below does everything we need